src/services/db/GitFileSystemService.ts

```diff
@@ -479,6 +483,50 @@ export default class GitFileSystemService {
   // TODO: Delete a file
   async delete() {}
 
-  // TODO: Get the latest commit of branch
-  async getLatestCommitOfBranch() {}
+  isGitLocalDiskRawCommitData(
+    commit: unknown
+  ): commit is GitLocalDiskRawCommitData {
+    return (
+      !!commit &&
```

Comment:
Should we also check to make sure `commit` is an object before proceeding to avoid unexpected issue? 
```ts
!!commit && typeof commit === `object` && {rest}
```

Reply:
sure!